sys/linux/dev_vhci.txt

```diff
@@ -40,7 +44,7 @@ vhci_command_vendor_pkt {
 vhci_vendor_pkt_opcode = HCI_PRIMARY, HCI_AMP, HCI_EXTERNAL_CONFIG, HCI_RAW_DEVICE
 
 type hci_acl_hdr[DATA] {
-	handle	const[0x100, int16:12]
+	handle	const[200, int16:12]
```

Comment:
Use hci_conn_handle here.

Reply:
Ah nevermind, hci_conn_handle is a type.

Reply:
This won't because of the bitfield. hci_conn_handle already encodes full int16 type...